### PR TITLE
Customizable From header

### PR DESCRIPTION
**Pull request by bajansen**
Customizable from header in advanced settings for password restore mail.

### CODE REVIEW

**Comment by EugeneKay** [comment]
This looks great, thank you. I will test + merge later tonight.

File: libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_nl.properties [hunk 1]
```diff
@@ -343,8 +343,9 @@ advancedsettings.smtpPort = SMTP poort
 advancedsettings.smtpServer = SMTP server
 advancedsettings.smtpEncryption = SMTP versleutelingsmethode
 advancedsettings.smtpUser = SMTP gebruikersnaam
-advancedsettings.smtpPassword = SMTP wachtwoord
+advancedsettings.smtpPassword = Wachtwoord
```
Comment (stevenroose) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_nl.properties]: Why did you change this? It is the SMPT password that is asked here, right?

Comment (bajansen) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_nl.properties]: Yup, but as I have rearranged the interface so the password field is now located behind the SMTP User field instead of underneath it, just putting 'Password' or in Dutch, 'Wachtwoord', looks cleaner than 'SMTP Password'. The password field for the LDAP Manager is also only titled 'Password'.

Comment (stevenroose) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_nl.properties]: Is there a reason why this is only changed in the Dutch language?

Comment (bajansen) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_nl.properties]: It's also changed in English, but the English language file also has some help text additions I haven't translated to Dutch so maybe that's why you missed it.

Comment (stevenroose) [inline libresonic-main/src/main/resources/org/libresonic/player/i18n/ResourceBundle_nl.properties]: Aha, I see it. We should hook Libresonic up with a service like Transifex or something :p
